### PR TITLE
remove leftover legacy code, rename logentries to insight in related places

### PR DESCRIPTION
**Pull request by stopal-r7**
This aligns some config keys with documentation where it was changed to Insight from Logentries but the code part was overlooked. Also removes some leftovers of legacy features in code.

@snakefoot can you also review?

### CODE REVIEW

**Comment by snakefoot** [comment]
Can only say that I would prefer my PR to be merged first. Then you can follow up with whatever cleanup PR's that you have been holding back for the last year.


Sent from my Samsung device


-------- Original message --------

**Comment by stopal-r7** [comment]
@snakefoot yeah, already merged

File: src/R7InsightCore/AsyncLogger.cs [hunk 1]
```diff
@@ -37,21 +37,13 @@ public class AsyncLogger
...
 
         // Error message displayed when invalid token is detected. 
-        protected const String InvalidTokenMessage = "\n\nIt appears your LOGENTRIES_TOKEN value is invalid or missing.\n\n";
-
-        // Error message displayed when invalid account_key or location parameters are detected. 
-        protected const String InvalidHttpPutCredentialsMessage = "\n\nIt appears your LOGENTRIES_ACCOUNT_KEY or LOGENTRIES_LOCATION values are invalid or missing.\n\n";
+        protected const String InvalidTokenMessage = "\n\nIt appears your log token value is invalid or missing.\n\n";
 
         // Error message deisplayed when queue overflow occurs. 
         protected const String QueueOverflowMessage = "\n\nLogentries buffer queue overflow. Message dropped.\n\n";
```
Comment (danilosterrapid7) [inline src/R7InsightCore/AsyncLogger.cs]: Logentries -> Insight

**Comment by danilosterrapid7** [comment]
There are also references to Logentries in `src/R7InsightCore/IAsyncLoggerConfig.cs` in comments and as `LOGENTRIES_TOKEN`

**Pull request merged**